target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -154,7 +154,8 @@ wavlink,wl-wn578a2)
 	ucidef_set_led_switch "wan" "wan" "green:wan" "switch0" "0x10"
 	;;
 xiaomi,mi-router-4a-100m|\
-xiaomi,mi-router-4a-100m-intl)
+xiaomi,mi-router-4a-100m-intl\
```

Comment:
Missing |, should by | \

Reply:
Goddammit, I was certain I had those there. Well, fixed.